### PR TITLE
feat: add settings code gen

### PR DESCRIPTION
**Pull request by Oshioke-Salaki**

### CODE REVIEW

File: app/components/page.tsx [hunk 1]
```diff
@@ -20,11 +21,9 @@ const Components = () => {
...
         <Preview
           name="Settings"
-          variants={["list", "grid"]}
-          codeStringGenerator={connCodeGen}
+          variants={[]}
+          codeStringGenerator={settingsCodeGen}
           description="lorem Lorem ipsum dolor sit amet consectetur adipisicing elit. Minima exercitationem rem quos, at consequuntur magni blanditiis quae libero debitis repudiandae dolorum dicta amet explicabo nihil? Id voluptas nihil culpa! Reiciendis."
-          themeVariants={false}
-          layoutVariants={false}
         >
```
Comment (stephanniegb) [inline app/components/page.tsx:27]: Why did you delete these?

Comment (Oshioke-Salaki) [reply app/components/page.tsx:27]: Because the settings doesn't make use of theme and layout prop

**Review by stephanniegb — APPROVED** [review]
